xml/System/Array.xml

```diff
@@ -6416,10 +6416,14 @@ int[,,] TDArray = new int[1,1,1];
         <Parameter Name="array" Type="T[]" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="array">To be added.</param>
-        <summary>To be added.</summary>
+        <typeparam name="T">The type of the elements in <paramref name="array" />.</typeparam>
+        <param name="array">The one-dimensional <see cref="T:System.Array" /> of elements of type <typeparamref name="T" /> to reverse.</param>
```

Comment:
```suggestion
        <param name="array">The one-dimensional array of elements to reverse.</param>
```

Reply:
No, we need to disambiguate with the original methods. They should not have the same description.

Reply:
I don't think you need to disambiguate in a parameter

Reply:
No need. Usually we need to disambiguate the summaries so people can choose which one to pick.

Reply:
Should we use _one-dimensional generic array_ like David suggested below?

Reply:
Oh, it's not a summary. Nevermind.